Library/Formula/vault.rb

```diff
@@ -0,0 +1,35 @@
+require "language/go"
+
+class Vault < Formula
+  homepage "https://vaultproject.io/"
+  version "0.1.0"
+  url "https://github.com/hashicorp/vault.git"
```

Comment:
You need to actually use a tagged release, not just the master branch. The ` head ` points at this, the ` url ` pretty much never should. Use ` https://github.com/hashicorp/vault/archive/v0.1.0.tar.gz ` instead.